### ISSUE TITLE
can you add loss  for confidence intervals for example quantile regression 

### ISSUE THREAD

**Issue by Sandy4321** [Mayurji/MLWithPytorch#4]
can you add loss  for confidence intervals for example quantile regression

**Comment by Mayurji** [comment]
@Sandy4321 , Can you elaborate on what is that you're asking?  Can you share any links?

**Comment by Sandy4321** [comment]
sure
https://towardsdatascience.com/quantile-regression-from-linear-models-to-trees-to-deep-learning-af3738b527c3

https://colab.research.google.com/drive/1nXOlrmVHqCHiixqiMF6H8LSciz583_W2#scrollTo=esTAKyTyG1TS

https://www.statsmodels.org/dev/examples/notebooks/generated/quantile_regression.html

https://towardsdatascience.com/deep-quantile-regression-c85481548b5a

https://towardsdatascience.com/deep-quantile-regression-in-tensorflow-1dbc792fe597

especially mape loss for quantile
https://github.com/keras-team/keras/blob/2c48a3b38b6b6139be2da501982fd2f61d7d48fe/keras/losses.py#L372-L429
https://github.com/scikit-learn/scikit-learn/blob/0d378913b/sklearn/metrics/_regression.py#L286

https://github.com/keras-team/keras/issues/15599

**Comment by Sandy4321** [comment]
https://pytorch-forecasting.readthedocs.io/en/latest/api/pytorch_forecasting.metrics.QuantileLoss.html

https://www.kaggle.com/carlossouza/quantile-regression-pytorch-tabular-data-only

https://github.com/pytorch/pytorch/issues/38035

https://github.com/maxmarketit/Auto-PyTorch/blob/develop/examples/quantiles/Quantiles.ipynb

https://arxiv.org/abs/1811.00908v3
